App/Settings/Settings.plist

```diff
@@ -115,6 +115,16 @@
 					<key>Key</key>
 					<string>embed_tweets</string>
 				</dict>
+				<dict>
+					<key>Title</key>
+					<string>Disable timg when posting</string>
```

Comment:
This looks good overall. Let’s reword the title too to be affirmative. I’m also thinking we could move it into its own section of settings titled "Posting", and then we can remove that part from the title text as it’s implied? Something like:

**Posting**
Automatic [timg] ✅

Does that make sense?

Reply:
Yup makes sense. Will do 👍